compendium/tools/SCRCodeGen/ComponentInfo.cpp

```diff
@@ -99,13 +101,13 @@ namespace codegen
         std::string
         GetReferenceBinderStr(ReferenceInfo const& ref, bool injectReferences)
         {
-            auto isStatic = (ref.policy == "static");
+            auto isDynamic = (ref.policy == "dynamic");
```

Comment:
I think there are good arguments either way for making a temporary boolean like this or not, but I decided against it because it's not a pattern I saw used elsewhere in this part of the code and I wanted to keep things consistent.

Reply:
It doesn't look like you decided against it. It looks like you did it here in ComponentInfo.cpp. My only point is if we are going to do it here I would like to see it above as well

Reply:
This is an old version of the diff. I just pushed a commit that removes this conditional entirely because it is redundant.